From 493fb7017ee5ef68cd0b481f8285e16826765148 Mon Sep 17 00:00:00 2001
From: Xinyu Chen <xinyu.chen@freescale.com>
Date: Thu, 13 Sep 2012 15:38:31 +0800
Subject: [PATCH 338/486] ENGR00221840-5 jb upgrade: correct the HAL suffix

Change the HAL module's suffix to proper name.
alsa and light should be platform level HAL,
gps and sensors should be board level HAL.

Change-Id: I4f6f08cc4f4296386956a32c0b5e1b517c0e8071
Signed-off-by: Xinyu Chen <xinyu.chen@freescale.com>
---
 alsa/Android.mk       | 6 +++---
 libgps/Android.mk     | 2 +-
 libsensors/Android.mk | 2 +-
 lights/Android.mk     | 2 +-
 4 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/alsa/Android.mk b/alsa/Android.mk
index 3be4871..4088a31 100755
--- a/alsa/Android.mk
+++ b/alsa/Android.mk
@@ -36,7 +36,7 @@ ifeq ($(strip $(BOARD_USES_ALSA_AUDIO)),true)
   	liblog   \
     libcutils
 
-  LOCAL_MODULE:= alsa.freescale
+  LOCAL_MODULE:= alsa.$(TARGET_BOARD_PLATFORM)
 
   LOCAL_MODULE_TAGS := eng
 
@@ -47,7 +47,7 @@ endif
 
 
 include $(CLEAR_VARS)
-LOCAL_MODULE := audio.primary.freescale
+LOCAL_MODULE := audio.primary.$(TARGET_BOARD_PLATFORM)
 LOCAL_MODULE_PATH := $(TARGET_OUT_SHARED_LIBRARIES)/hw
 LOCAL_SRC_FILES := imx_audio_hal.cpp
 LOCAL_SHARED_LIBRARIES := liblog libcutils libutils libhardware
@@ -55,7 +55,7 @@ LOCAL_MODULE_TAGS := optional
 include $(BUILD_SHARED_LIBRARY)
 
 include $(CLEAR_VARS)
-LOCAL_MODULE := audio.tinyalsa.freescale
+LOCAL_MODULE := audio.tinyalsa.$(TARGET_BOARD_PLATFORM)
 LOCAL_MODULE_PATH := $(TARGET_OUT_SHARED_LIBRARIES)/hw
 LOCAL_SRC_FILES := tinyalsa_hal.c
 LOCAL_C_INCLUDES += \
diff --git a/libgps/Android.mk b/libgps/Android.mk
index 06a0a53..f92fecc 100644
--- a/libgps/Android.mk
+++ b/libgps/Android.mk
@@ -9,7 +9,7 @@ ifneq ($(TARGET_PRODUCT),sim)
         LOCAL_SHARED_LIBRARIES := liblog libcutils libhardware
         LOCAL_SRC_FILES := athr_gps.c
         LOCAL_SRC_FILES += gps.c
-        LOCAL_MODULE := gps.freescale
+        LOCAL_MODULE := gps.$(TARGET_BOOTLOADER_BOARD_NAME)
         LOCAL_MODULE_TAGS := optional
         include $(BUILD_SHARED_LIBRARY)
     endif
diff --git a/libsensors/Android.mk b/libsensors/Android.mk
index 7db222c..3116a09 100755
--- a/libsensors/Android.mk
+++ b/libsensors/Android.mk
@@ -22,7 +22,7 @@ ifneq ($(TARGET_SIMULATOR),true)
 include $(CLEAR_VARS)
 LOCAL_PRELINK_MODULE := false
 LOCAL_MODULE_PATH := $(TARGET_OUT_SHARED_LIBRARIES)/hw
-LOCAL_MODULE := sensors.freescale
+LOCAL_MODULE := sensors.$(TARGET_BOOTLOADER_BOARD_NAME)
 LOCAL_MODULE_TAGS := eng
 
 LOCAL_CFLAGS := -DLOG_TAG=\"Sensors\"
diff --git a/lights/Android.mk b/lights/Android.mk
index 54d5cd5..d01e5c8 100644
--- a/lights/Android.mk
+++ b/lights/Android.mk
@@ -21,6 +21,6 @@ LOCAL_PRELINK_MODULE := true
 LOCAL_MODULE_PATH := $(TARGET_OUT_SHARED_LIBRARIES)/hw
 LOCAL_SHARED_LIBRARIES := liblog libcutils
 LOCAL_SRC_FILES := fsl_lights.c
-LOCAL_MODULE := lights.freescale
+LOCAL_MODULE := lights.$(TARGET_BOARD_PLATFORM)
 LOCAL_MODULE_TAGS := eng
 include $(BUILD_SHARED_LIBRARY)
-- 
1.8.0

